.env.example

```
@@ -0,0 +1,2 @@
+MESSAGES_SERVICE_URL = 'http://localhost:81'
+AUTH_SERVICE_URL = 'http://localhost:82'
```

```suggestion
AUTH_SERVICE_URL = 'http://localhost:82'

```
Добавить перенос строки в конце файла